sdgym/datasets.py

```diff
@@ -69,9 +86,11 @@ def load_dataset(dataset, datasets_path=None, bucket=None, aws_key=None, aws_sec
     return metadata
 
 
-def load_tables(metadata):
+def load_tables(metadata, max_rows=None):
     real_data = metadata.load_tables()
     for table_name, table in real_data.items():
+        if max_rows and table.shape[0] > max_rows:
```

Comment:
Same as above about allowing `max_columns` only if this is a single table datasets: If there are more than one table and max_rows is defined, let's raise a ValueError saying that it is not supported for multi-table datasets.